main.js

```diff
@@ -0,0 +1,240 @@
+  let diagR = cell2.textContent+cell4.textContent+cell6.textContent;
+  let checkAnswer = [row1, row2, row3, col1, col2, col3, diagL, diagR]; //All the div textContent for the winning combinations added to array for iteration
+  let lines = ["url('images/TopRow.png')","url('images/MidRow.png')","url('images/BotRow.png')","url('images/LeftCol.png')", "url('images/CenterCol.png')", "url('images/RightCol.png')", "url('images/LeftDiag.png')", "url('images/RightDiag.png')"];
+  
+  checkAnswer.forEach((cond)=>{
+    if(cond === "XXX"){
+      if(player1Name){
+       status.textContent = `${player1Name}: X's win!`;
+      }else{
+        status.textContent = `X's win!`;
+      }
+      gameOver = true;
+      gameStarted = false;
+      buttonArr.forEach((button)=> button.disabled = false);
+      board.style.backgroundImage = lines[checkAnswer.indexOf(cond)]; //I could change the lines array to an object with the win condition as the key and the image as the value
+    }
+    if(cond === "OOO"){
+      if(player2Name){
+        status.textContent = `${player2Name}: O's win!`;
+       }else{
+         status.textContent = `O's win!`;
+       }
+      gameOver = true;
+      gameStarted = false;
+      buttonArr.forEach((button)=> button.disabled = false);
+      board.style.backgroundImage = lines[checkAnswer.indexOf(cond)];
+    }
+   
+  })
+  if(gameStarted && allCells.every(element => element.textContent !== "")){
+    status.textContent = `It's a draw!`;
+    gameOver = true;
+    gameStarted = false;
+    buttonArr.forEach((button)=> button.disabled = false);
+  }
+}
+
+function timerFunction(){
+  let seconds = 0;
+  let mins = 0;
+  let hours = 0;
+  let gameTime = window.setInterval(countSecs, 1000);
+  let timer = document.getElementById('timer');
+  function countSecs(){
+    if(gameOver){
+      clearInterval(gameTime);
+    }else{seconds++;}
+    if(seconds==60){ 
+      mins++;
+      seconds = 0;
+    }
+    if(mins==60){
+      hours++;
+      mins = 0;
+    } 
+    timer.textContent = `Time Elapsed: ${hours.toString().padStart(2,"0")}:${mins.toString().padStart(2,"0")}:${seconds.toString().padStart(2,"0")}`;
+  }
+}
+
+function initializeGame(){
+  allCells.forEach(cell => cell.textContent = "");
+  gameStarted = true;
+  gameOver = false;
+  board.style.backgroundImage = null;
+  return (numOfPlayers > 0 ? whosTurn = 'player1' : whosTurn = 'computer1');
+}
+
+function computerPicks() { 
+  let emptyCells = [];
+  for(let i = 0; i<allCells.length; i++){
+    if(allCells[i].textContent === ""){
+      emptyCells.push(allCells[i]);
+    }
+  }
+  let compChoice = Math.floor(Math.random() * Math.floor(emptyCells.length))
+  if(whosTurn === 'computer1'){
+    emptyCells[compChoice].textContent = "X";
+  }else{
+    emptyCells[compChoice].textContent = "O";
+  }
+  checkforWinner();
+  if(!gameOver){switchTurn()};
+  if(!gameOver && numOfPlayers === 0){
+    setTimeout(computerPicks, 500);
+  }
```

Comment:
All is well and neat in your javascript file. It seems like you've completed the backlog. I liked your choice of styling when it comes to modifying the CSS file. Asides from that,